ui/src/main/resources/PhenomeCentral/RemoteMatching.xml

```diff
@@ -491,16 +486,14 @@ td input[type="text"]:focus {
             #set ($serverName = $serverID)
           #end
           #set ($resultsURL = $xwiki.getURL('PhenomeCentral.RemoteResults', 'get', ""))
-          (% class="similarity-mme-server-name-header" id="$headerID" %)((({{html clean=false}}&lt;h4&gt;$serverName&lt;/h4&gt;
           #set ($disclaimer = $config.getProperty('disclaimer').getValue())
-          #if ($disclaimer &amp;&amp; $disclaimer != "")
-            &lt;div class="hint mme-disclaimer"&gt;$disclaimer&lt;/div&gt;
-          #end
-          {{/html}})))##
+          {{html clean=false}}##
+          &lt;h4 class="similarity-mme-server-name-header" id="$headerID"&gt;$serverName&lt;/h4&gt;
+          {{/html}}##
           (% class="similarity-results-container similarity-results-container-mme" id="similar-remote-patients-container-$serverName" %)((({{html clean=false}}
-            &lt;input type="hidden" name="refreshButtonContainer" value="$headerID"/&gt;
             &lt;input type="hidden" name="service" value="$resultsURL"/&gt;
-            &lt;input type="hidden" name="mmeServer" value="$serverID"/&gt;{{/html}})))##
+            &lt;input type="hidden" name="mmeServer" value="$serverID"/&gt;
+            &lt;input type="hidden" name="serviceDisclaimer" value="$disclaimer"/&gt;{{/html}})))##
```

Comment:
Exactly, it is put inside an attribute, which means that any quotes will break it. If it were printed outside, it could be put as-is, but as an attribute, it must be escaped. Correct?